lib/search_files.js

```diff
@@ -0,0 +1,291 @@
+var fs = require('fs');
+var os = require('os');
+var path = require('path');
+var glob = require('glob');
+
+
+function isWindows() {
+  return process.platform === 'win32';
+}
+
+function isFunction(v) {
+  return (typeof v === 'function');
+}
+
+function isObject(v) {
```

Comment:
@sttk thanks for looking into that.  I'm fine with the npm modules (`lodash.mapvalues`, etc).  Please note that isObject is not the same as isPlainObject, lodash actually has 3 methods for objects (isPlainObject, isObject and isObjectLike)